frontend/src/store/root-schema.ts

```diff
@@ -1,3 +1,4 @@
+import { Stats } from "../models/stats.model";
 import { GameState } from "../models/game-state.model";
 
 export interface RootSchema {
```

Comment:
You should add "stats" to `state` definition

Reply:
I don't understand what you mean...

Reply:
Take a look at RoomSchema, and field `state`. It should be `"login" | "lobby" | "game" | "stats"` now, should it not?